Assets/_Scripts/FieldState.cs

```diff
@@ -3,101 +3,34 @@
 using UnityEngine;
 using UnityEngine.UI;
 using System.Linq;
-using UnityEngine.SceneManagement;
 
+// Class that holds field information / helper functions
 public class FieldState : MonoBehaviour
-{
-    private Enemy[] _enemies = new Enemy[Party.numPlayers];
+{   
+    public Party<Player> playerParty = new Party<Player>();
+    public Party<Enemy> enemyParty = new Party<Enemy>(); 
 
 
     [Header("Slots")]
-    public Transform heroSlot;
-    public List<Transform> mookSlots;
-    public List<Transform> enemySlots;
+    public FighterSlot heroSlot;
+    public List<FighterSlot> mookSlots;
+    public List<FighterSlot> enemySlots;
 
```

Comment:
Do we really need to store information about both the List of FightingEntities and the Slots? Keeping track of the Slots should also keep track of the FightingEntities in these slots.

Reply:
The Slots are just for the initialization locations, so yes.